provider_counter/test/widget_test.dart

```diff
@@ -1,14 +1,14 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
-import 'package:scoped_model/scoped_model.dart';
-import 'package:scoped_model_counter/main.dart';
+import 'package:provider/provider.dart';
+import 'package:provider_counter/main.dart';
 
 void main() {
   testWidgets('Counter increments smoke test', (WidgetTester tester) async {
     // Build our app, provide it with a model, and trigger a frame.
     await tester.pumpWidget(
-      ScopedModel(
-        model: Counter(),
+      ChangeNotifierProvider(
+        builder: (_) => Counter(),
```

Comment:
Same nit here as in the other PR. Just mentioning it again so it doesn't get overwritten by this PR. 😄